includes/wikia/models/DesignSystemGlobalFooterModel.class.php

```diff
@@ -188,202 +196,350 @@ class DesignSystemGlobalFooterModel extends WikiaModel {
+				[
+					'type' => 'link-text',
+					'title' => [
+						'type' => 'translatable-text',
+						'key' => 'global-footer-wikia-community-link-help'
+					],
+					'href-key' => 'help'
 				]
+			]
+		],
+		'create_wiki' => [
+			'description' => [
+				'type' => 'translatable-text',
+				'key' => 'global-footer-wikia-create-wiki-description'
 			],
-			'licensing_and_vertical' => [
-				'description' => [
+			'links' => [
+				[
+					'type' => 'link-text',
+					'title' => [
+						'type' => 'translatable-text',
+						'key' => 'global-footer-wikia-create-wiki-link-start-wikia'
+					],
+					'href-key' => 'create-new-wiki'
+				]
+			]
+		],
+		'community_apps' => [
+			'header' => [
+				'type' => 'line-text',
+				'title' => [
 					'type' => 'translatable-text',
-					'key' => 'global-footer-licensing-description',
-					'params' => [
-						'license' => $this->getLicenseData(),
-					]
+					'key' => 'global-footer-wikia-community-apps-header'
+				]
+			],
+			'description' => [
+				'type' => 'translatable-text',
+				'key' => 'global-footer-wikia-community-apps-description'
+			],
+			'links' => [
+				[
+					'type' => 'link-image',
+					'image' => 'wds-company-store-appstore',
+					'title' => [
+						'type' => 'translatable-text',
+						'key' => 'global-footer-wikia-community-apps-link-app-store'
+					],
+					'href-key' => 'app-store'
 				],
+				[
+					'type' => 'link-image',
+					'image' => 'wds-company-store-googleplay',
+					'title' => [
+						'type' => 'translatable-text',
+						'key' => 'global-footer-wikia-community-apps-link-google-play'
+					],
+					'href-key' => 'google-play'
+				]
+			]
+		],
+		'advertise' => [
+			'header' => [
+				'type' => 'line-text',
+				'title' => [
+					'type' => 'translatable-text',
+					'key' => 'global-footer-wikia-advertise-header'
+				]
 			],
-		];
+			'links' => [
+				[
+					'type' => 'link-text',
+					'title' => [
+						'type' => 'translatable-text',
+						'key' => 'global-footer-wikia-advertise-link-media-kit'
+					],
+					'href-key' => 'media-kit'
+				],
+				[
+					'type' => 'link-text',
+					'title' => [
+						'type' => 'translatable-text',
+						'key' => 'global-footer-wikia-advertise-link-contact'
+					],
+					'href-key' => 'media-kit-contact'
+				]
+			]
+		],
+	];
+
+	private $data = [ ];
+
+	private $wikiId;
+	private $lang;
+
+	public function __construct( $wikiId, $lang = self::DEFAULT_LANG ) {
+		parent::__construct();
+
+		$this->wikiId = $wikiId;
+		$this->lang = $lang;
+	}
 
-		$data['fandom_overview'] = $this->getFandomOverview();
-		$data['follow_us'] = $this->getFollowUs();
-		$data['community'] = $this->getCommunity();
+	public function setHrefs( $hrefs ) {
+		$this->hrefs = $hrefs;
+	}
 
+	public function getData() {
+		$this->setHeaderData();
+
+		foreach ( $this->baseData as $sectionName => $sectionBaseData ) {
+			$this->setSectionData( $sectionName, $sectionBaseData );
+		}
+
+		$this->setLicenseData();
+
+		return $this->data;
```

Comment:
this could be more readable if you have:
- $this -> setHeaderData();
- $schemaData = $this -> getSchemaData();
- return setOutputData($schemaData);

setOutputData could have:
- setSectionData()
- setLicenseData()